src/index.html

```diff
@@ -9,11 +9,31 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+  <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+<div class="product_card" data-qa="card">
+  <img class="product_image" src="./images/imac.jpeg" alt="">
+  <p class="product_name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```

Comment:
It is a header.

Reply:
sorry, our mistake, Dima meant heading tag